src/pages/try.js

```diff
@@ -57,7 +69,23 @@ export default class Try extends Component {
     jsIsLatest: false,
     autoEvaluate: true,
     output: [],
+    tryUrl: window.location.href,
+  }
+
+  handleClick = this.handleClick.bind(this);
+
+   handleClick() {
+    var val = this.encodeBase64(this.state.reason);
+    this.setState({tryUrl: val});
+    document.getElementById('digest').disabled = false;
```

Comment:
using IDs & getElementById is super not reacty :/
instead of having that input, could we just do `window.location = val`? Then ppl can copy/paste the url from their browser